tools/recover/README.md

```diff
@@ -0,0 +1,27 @@
+# State Recover
+
+The Binance Chain has a hardfork upgrade and if you failed to upgrade your fullnode to the latest version, `bnbchaind` process will stop and if you restart with the latest version, the following error will appear:
```

Comment:
it will stop if you run the node with the old binary, right, not the latest verison